server/core/region_option.go

```diff
@@ -179,6 +190,18 @@ func SetReadKeys(v uint64) RegionCreateOption {
 	}
 }
 
+// SetReadQueryNum sets the read query num for the region.
+func SetReadQueryNum(v uint64) RegionCreateOption {
```

Comment:
I think this function is hard to be used in another test. It is too specific.

Reply:
update